lib/console/deployments/pr/impl/bitbucket.ex

```diff
@@ -63,7 +69,37 @@ defmodule Console.Deployments.Pr.Impl.BitBucket do
     end
   end
 
-  def files(_, _), do: {:ok, []}
+  def files(conn, url) do
+    with {:ok, workspace, repo, pr_id} <- get_pull_id(url),
+         {:ok, pr_info} <- get_pr_info(conn, workspace, repo, pr_id),
+         diff_url = pr_info["links"]["diff"]["href"],
```

Comment:
I'd honestly extract these in the match at line 74 too (would technically be safer as well, eg:

```elixir
{:ok, %{"links" => %{"diff" => %{"href" => diff}, "diffstat" => %{"href" => diffstat}}} <- ...
```

(or better yet, do that in line within the function body of `get_pr_info` and return a tuple like {:ok, info, diff_url, diffstat_url}